### PR TITLE
Update SIG Testing zoom meetings

### LINKED ISSUE #5129: Update all SIG Testing zoom meetings to use password
Per chairs-and-TLs meeting http://bit.ly/k8s-chair-tl-notes

- [x] kind
- [x] testing-commons
- [x] sig-testing

/assign
/sig testing
/sig contributor-experience

### PR DESCRIPTION
**Pull request by spiffxp**
Fixes #5129

/cc @BenTheElder @stevekuznetsov

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2280,7 +2271,7 @@ sigs:
       time: "13:00"
       tz: PT (Pacific Time)
       frequency: bi-weekly
-      url: https://zoom.us/j/790505720
+      url: https://zoom.us/j/790505720?pwd=VThEeVJqTnFPY0o1bDFZQk1ZRjJLQT09
       archive_url: https://docs.google.com/document/d/1TOC8vnmlkWw6HRNHoe5xSv5-qv7LelX6XK3UVCHuwb0/edit
```
Comment (SergeyKanzhelev) [inline sigs.yaml:2275]: nit: recordings_url? same as test bi-weekly meetings

**Comment by BenTheElder** [comment]
LGTM

**Comment by mrbobbytables** [comment]
Linking to tracking issue: https://github.com/kubernetes/community/issues/5146

**Pull request merged**